pvlib/pvsystem.py

```diff
@@ -977,6 +984,18 @@ def __init__(self, mount,
         else:
             self.temperature_model_parameters = temperature_model_parameters
 
+        if self.temperature_model_parameters=={}:
```

Comment:
@matsuobasho I agree with @kandersolar point of view, that we should remove this `ValueError`. I think we'll want to undo most of the changes in test_pvsystem.py as well. Because I can push to your fork, let me know if you'd like some help. And thanks again for your effort here.

Reply:
It'll be most efficient to delete this entire MR because there are so many changes that will not be used.  The proposed change is just deleting two lines and will be easiest to recreate with a new MR.

Let me know if you can kill the entire MR.  I'll then create a new MR with just that one change.  

In this case, it looks like we do not need a new test or any docstring modifications?

Reply:
You can close this PR and start a new one.

For the new PR I recommend that you first create a branch in your fork and use that branch for changes. Its not usually a good idea to make changes in your main branch.

First reset your main branch

git reset --hard upstream/ main   where upstream is your name for pvlib/pvlib-python

Then create a branch

git checkout -b <branch name>

Make the edits in the branch, add, commit, push, etc.